pkg/electron/electronDownloader.go

```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/develar/app-builder/pkg/log"
 	"github.com/develar/app-builder/pkg/util"
 	"github.com/develar/errors"
-	"github.com/develar/go-fs-util"
-	"github.com/json-iterator/go"
+	fsutil "github.com/develar/go-fs-util"
+	jsoniter "github.com/json-iterator/go"
```

Comment:
Superfluous changes?

Reply:
I think this is added by `go fmt`, so should I remove this?